field_select.go

```diff
@@ -361,16 +521,22 @@ func (s *Select[T]) titleView() string {
 }
 
 func (s *Select[T]) descriptionView() string {
-	return s.activeStyles().Description.Render(s.description)
+	return s.activeStyles().Description.Render(s.description.val)
 }
 
-func (s *Select[T]) choicesView() string {
+func (s *Select[T]) optionsView() string {
 	var (
 		styles = s.activeStyles()
 		c      = styles.SelectSelector.String()
 		sb     strings.Builder
 	)
 
+	if s.options.loading && time.Since(s.options.loadingStart) > spinnerShowThreshold {
+		s.spinner.Style = s.activeStyles().MultiSelectSelector.UnsetString()
+		sb.WriteString(s.spinner.View() + " Loading...")
```

Comment:
Hi @maaslalani, I noticed that the ‘Loading’ message is currently hardcoded. It might be beneficial to allow this message to be defined dynamically, perhaps with a method like ‘LoadingMessage()’. While this isn’t critical for functionality, it could be quite useful for customization and internationalization.

Reply:
Hey @OliveiraCleidson, thanks for pointing this out. We can fix this in a future release! Great call.